executor/builder.go

```diff
@@ -715,12 +715,6 @@ func (b *executorBuilder) buildSort(v *plan.Sort) Executor {
 		schema:       v.Schema(),
 	}
 	sortExec.supportChk = true
-	if v.ExecLimit != nil {
```

Comment:
Why not consider ExecLimit field?

Reply:
This branch never runs.
If it has LIMIT, TopN will be built instead